README.md

```diff
@@ -528,6 +529,13 @@ Another alternative is [Chuck](https://github.com/jgilfelt/chuck) which, althoug
 
 Antti Lammi, Joni Karppinen, Peter Tackage, Timo Tuominen, Vera Izrailit, Vihtori Mäntylä, Mark Voit, Andre Medeiros, Paul Houghton and other Futurice developers for sharing their knowledge on Android development.
 
+# Acknowledgements
+
+This project is sponsored by [Spice Program](http://spiceprogram.org/chilicorn-history/), our open source and social impact program made with love by [Futurice](http://www.futurice.com).
+
+[![Chilicorn Logo](https://raw.githubusercontent.com/futurice/spiceprogram/gh-pages/assets/img/logo/chilicorn_no_text-256.png)](http://spiceprogram.org/chilicorn-history/)
+Brought to you by the power of the Chilicorn!
```

Comment:
It probably looks better without "Brought to you by the power of the Chilicorn!". Otherwise it sounds a little repetitious.